src/controllers/addresses.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable radix */
+/* eslint-disable no-console */
+/* eslint-disable consistent-return */
+import connection from '../database/database.js';
+import { addressFactory } from '../factories/addresses.factory.js';
+import { validateAddress } from '../validation/addresses.js';
+
+async function getAddresses(req, res) {
+    const token = req.headers.authorization?.split('Bearer ')[1];
+
+    try {
+        const addresses = await connection.query(`
+            SELECT 
+                addresses.id, addresses.address, addresses.postal_code, addresses.comp
+            FROM 
+                sessions  
+            JOIN 
+                addresses 
+            ON 
+                sessions.user_id = addresses.user_id
+            WHERE 
+                token = $1;`,
```

Comment:
É uma boa pratica definir à qual tabela aquela coluna pertence. Nesse caso é apenas um detalhe para Refactor.

```suggestion
                sessions.token = $1;`,
```